help/OpenMP_mac.md

```diff
@@ -0,0 +1,30 @@
+## Installing OpenMP on Mac OSX
+
+In some cases, you may get an error that does not allow installation of goldfish from source on Mac OSX versions, including under R 4.0.0. The error may relate to compiling the parts of goldfish that are written in C++, or whether OpenMP (for parallelisation) can be found. To install OpenMP, follow these steps:
+
+1. Install [homebrew](https://brew.sh/) via the terminal. Homebrew allows you to easily install software.
+2. Install the latest version of `gcc` via the terminal command `brew install gcc`. 
+3. Check what version of `gcc` you have with: `brew list --versions gcc`
+4. Go to `~/.R` and open the file `Makevars`. Add the following code to that file:
+```
```

Comment:
This is great, @timonelmer! I would suggest that you add a few empty lines between code fence and text though, and between text and list items (lines 24 and 25).

Please note that when merged this will trigger a new release, but I will then delete it so that we don't lose our versioning system.

Reply:
@jhollway : Seems the pull request is complete. Is it fine to merge it?

Reply:
Absolutely. Already approved (23/11/2020).